doc/README.build_feedstock.md

```diff
@@ -19,50 +19,56 @@ Command usage for the `build_feedstock.py` command:
-                        [--python_versions PYTHON_VERSIONS_LIST]
-                        [--build_types BUILD_TYPES_LIST]
-                        [--working_directory WORKING_DIRECTORY]
-                        [--channels CHANNELS_LIST]
+usage: build_feedstock.py [-h] [--conda_build_config CONDA_BUILD_CONFIG]
+                          [--output_folder OUTPUT_FOLDER]
+                          [--channels CHANNELS_LIST]
+                          [--python_versions PYTHON_VERSIONS]
+                          [--build_types BUILD_TYPES] [--mpi_types MPI_TYPES]
+                          [--recipe-config-file RECIPE_CONFIG_FILE]
+                          [--recipes RECIPE_LIST]
+                          [--working_directory WORKING_DIRECTORY]
+                          [--local_src_dir LOCAL_SRC_DIR]
 
 Build conda packages as part of Open-CE
 
 optional arguments:
   -h, --help            show this help message and exit
-  --recipe-config-file RECIPE_CONFIG_FILE
-                        Path to the recipe configuration YAML file. The configuration
-                        file lists paths to recipes to be built within a feedstock.
-
-                        Below is an example stating that there are two recipes to build,
-                        one named my_project and one named my_variant.
-
-                        recipes:
-                          - name : my_project
-                            path : recipe
-
-                          - name : my_variant
-                            path: variants
-
-                        If no path is given, the default value is `build-config.yaml`.
-                        If `build-config.yaml` does not exist, and no value is provided,
-                        it will be assumed there is a single recipe with the
-                        path of "recipe".
+  --conda_build_config CONDA_BUILD_CONFIG
+                        Location of conda_build_config.yaml file. (default:
+                        /mnt/pai/home/bnelson/git/open-ce/open-
+                        ce/../conda_build_config.yaml)
   --output_folder OUTPUT_FOLDER
                         Path where built conda packages will be saved.
+                        (default: condabuild)
+  --channels CHANNELS_LIST
+                        Conda channels to be used. (default: [])
+  --python_versions PYTHON_VERSIONS
+                        Comma delimited list of python versions to build for,
+                        such as "3.6" or "3.7". (default: 3.6)
+  --build_types BUILD_TYPES
+                        Comma delimited list of build types, such as "cpu" or
+                        "cuda". (default: cpu,cuda)
+  --mpi_types MPI_TYPES
+                        Comma delimited list of mpi types, such as "openmpi"
+                        or "system". (default: openmpi)
+  --recipe-config-file RECIPE_CONFIG_FILE
+                        Path to the recipe configuration YAML file. The
```

Comment:
This looks messy now.

Reply:
It looks clean on my end, what looks messy?
![image](https://user-images.githubusercontent.com/18013846/96011059-1b91cf80-0e08-11eb-96f9-170b786ae14a.png)
---
![image](https://user-images.githubusercontent.com/18013846/96011107-28aebe80-0e08-11eb-907a-58ebd07033a1.png)

Reply:
The previous section for recipe-config-file used to have new lines in it. This made the example YAML look readable.

Reply:
Haha, I obviously could have been a little more specific in the previous comment.